content/tbd/STRP.md

```diff
@@ -0,0 +1,38 @@
+---
+created: 2023-12-07
+contributors: Matthew McCarthy
+title: STRP
+url: https://strp.nl/
+locations:
+  - Eindhoven, NL
+start: "2006"
+end: 
+size: " 4-10"
+image: 
+sectors: Arts and Culture
+activities: Events and Retreats
+cohere: true
+metadata_version: "1"
+---
+
+# About
+
+STRP explores the relationship between art, technology, and major societal issues with the general public, artists, designers, media makers, and thinkers. The Eindhoven-based organisation starts dialogue about speculative future scenarios with various events throughout the year and an annual festival in the spring.
```

Comment:
We should always use "quotation marks" to flag where we are quoting directly from the website. i would say this is essential. and also link to where we are copying and pasting from.